ggshield/cmd/auth/utils.py

```diff
@@ -8,6 +8,10 @@
 
 VERSION_TOO_LOW_MESSAGE = "Your GitGuardian dashboard version is too low, upgrade to be able to use this command."
 DISABLED_FLOW_MESSAGE = "Your GitGuardian dashboard does not authorize this command, contact your administrator."
+CONNECTION_ERROR_MESSAGE = (
+    "Could not connect to GitGuardian.\n"
+    "Please check your internet connection or if the specified URL is correct."
```

Comment:
If I were to nit pick on logical conditions, I'd say you want to use a "and" connector instead of "or", since you want the user to check both conditions and not only one of them :angel:

Reply:
correct. My bad for the imperfection of my english ;)